README.md

```diff
@@ -7,32 +7,136 @@ Inspired from [Caddy](https://github.com/mholt/caddy).
 ## Features
 
 * Easy configuration with Maddyfile (same as Caddyfile)
-* Automatic TLS and DKIM support
 * Transparent PGP support
```

Comment:
I guess we can remove this as well. Might just remove the whole "Features" section altogether.

Reply:
Will remove Features section altogether.